eZ/Publish/Core/FieldType/Relation/Type.php

```diff
@@ -138,6 +150,35 @@ public function getName(SPIValue $value)
         throw new \RuntimeException('Name generation provided via NameableField set via "ezpublish.fieldType.nameable" service tag');
     }
 
+    /**
+     * Validates a field based on the validators in the field definition.
+     *
+     * @return \eZ\Publish\SPI\FieldType\ValidationError[]
+     */
+    public function validate(FieldDefinition $fieldDefinition, SPIValue $fieldValue)
+    {
+        $validationErrors = [];
+
+        if ($this->isEmptyValue($fieldValue)) {
+            return $validationErrors;
+        }
+
+        try {
+            $this->contentService->loadContent($fieldValue->destinationContentId);
```

Comment:
Should be lighter with the same effect.
```suggestion
            $this->contentService->loadContentInfo($fieldValue->destinationContentId);
```